packages/CanvasIFrameModule/readme.MD

```diff
@@ -0,0 +1,7 @@
+#Canvas iframe module
+
+This module creates a new module named Test that displays an iframe.  This module also creates a button that links to 
+the new Test module.  The button can be found toward the top of the Opportunities module.
+
+You MUST run a Quick Repair and Rebuild (can be found under Administration > Repair) in order for the button on the 
```

Comment:
We could add a post install/uninstall script that runs the equivalent repair so that Admins do not need to do this manually.  See https://github.com/sugarcrm/BuildingBlocks/blob/master/packages/ScriptLoader/ for an example.